routes/chat.js

```diff
@@ -72,4 +73,11 @@ router.post(
   SuccessMiddleware
 );
 
+router.post(
+  '/channel-detail',
+  auth.isAuthV2,
+  validate(ChatValidation.changeChannelDetail),
+  changeChannelDetail
+);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new endpoint `/channel-detail` is using `auth.isAuthV2` middleware for authentication. Please ensure that this middleware is correctly validating the user's authentication and authorization to update the channel details. If there are different roles in your application (like admin, user), you might need to add additional checks to ensure only authorized users can update the channel details.
```diff
router.post(
  '/channel-detail',
  auth.isAuthV2,
+   auth.isAuthorizedToUpdateChannel,
  validate(ChatValidation.changeChannelDetail),
  changeChannelDetail
);
```
In the above diff, `auth.isAuthorizedToUpdateChannel` is a hypothetical middleware function that you would need to implement if such role-based access control is required in your application.


<!-- This is an auto-generated comment by OSS CodeRabbit -->